spec/index.html

```diff
@@ -1399,7 +1399,7 @@ <h4>Dataset( <code><a data-cite="SPARQL12-QUERY#rQuadPattern">QuadPattern</a></c
               </li>
               <li>
                 <p><code><span class="token">'{'</span> <a data-cite="SPARQL12-QUERY#rTriplesTemplate">TriplesTemplate</a>? <span class="token">'}'</span></code></p>
-                <p>Dataset(<em>QuadPattern</em>, μ, DS, GS ) is the Dataset consisting of only a default graph composed by all <a data-cite="RDF11-CONCEPTS#dfn-triple">valid RDF
+                <p>Dataset(<em>QuadPattern</em>, μ, DS, GS ) is the Dataset consisting of only a default graph composed by all <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">valid RDF
```

Comment:
`Dataset(<em>QuadPattern</em>, μ, DS, GS )` should either have spaces wrapping both opening and closing parentheses, or neither. I prefer both, as below.
```suggestion
                <p>Dataset ( <em>QuadPattern</em>, μ, DS, GS ) is the Dataset consisting of only a default graph composed by all <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">valid RDF
```

Reply:
This PR is a fix needed for a specific link issue for FPWD.

Yes - `Dataset( ` is inconsistent - this isn't the only occurrence.

In the text, the more programming style (no spaces) works.. In the title, with spaces looks OK (which it already is for the wrong reason - there is a line break in the HTML).

Reply:
> This PR is a fix needed for a specific link issue for FPWD.

I'll make a fresh PR.

> In the text, the more programming style (no spaces) works.

That's debatable. I strongly believe (based on experience, which is unfortunately not citable research) that additional (technically optional) whitespace increases comprehensibility in text which is meant to educate and/or instruct the reader. People who are accustomed to reading punctuation-dense content without whitespace are not *usually* flummoxed by extra whitespace, while people who are not accustomed to reading such text find it *far* more comprehensible with that same extra whitespace.

Reply:
Style it.